templates/home/home.html.twig

```diff
@@ -45,6 +45,25 @@
                 </ul>
             </div>
         </div>
+
+        {% if latestImages is defined and latestImages is not empty %}
+            {{ 'Latest pictures'|trans }}:
+            <div class="container-fluid track_images">
```

Comment:
If you put images in container only, they are not centred and there is a black spot on the right.

You could use flex box for this. For example wrap the images inside `<div class="d-flex justify-content-between">`

Reply:
container-fluid is removed.
If I add d-flex then there is no space between the images.

Reply:
That's why you should add _justyfy-content-between_ or some margin.

Anyway, the images are still ugly, at least on my screen:
![Screenshot from 2019-09-16 23-42-31](https://user-images.githubusercontent.com/1723868/64992147-fc123580-d8db-11e9-9a17-39e0a39cbdbf.png)

Reply:
fixed